dashboard/modules/job/job_head.py

```diff
@@ -428,6 +432,12 @@ async def tail_job_logs(self, req: Request) -> Response:
 
         driver_agent_http_address = job.driver_agent_http_address
         driver_node_id = job.driver_node_id
+        if driver_agent_http_address is None:
+            return Response(
+                text="The driver process has not started running yet, "
+                "please try it later",
```

Comment:
```suggestion
                text="The entrypoint script has not started running yet, "
                "please try again later",
```
Nit, because we haven't exposed "driver process" in the jobs user docs, only "entrypoint script"

Reply:
thanks for the suggestion